packages/core/src/components/checkbox/test/disabled/checkbox.e2e.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/checkbox/test/disabled/index.html';
+
+test.describe('tds-checkbox', () => {
+  test('Hover over checkbox and label -> cursor should become inactive', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Find the checkbox and label elements
+    const checkbox = page.locator('tds-checkbox');
+    const labelElement = page.locator('tds-checkbox', { hasText: 'Label' });
```

Comment:
I think that here we need to target elements underneath, like `label` and `input` to check that the pointer is not allowed, otherwise, the test might be false positive.